packages/form-builder/src/components/FormFieldList/FormFieldList.vue

```diff
@@ -10,9 +10,14 @@
 			{{ listTitle }}
 		</h3>
 
-		<p v-if="description">
-			{{ description }}
-		</p>
+		<slot
+			name="description"
```

Comment:
```suggestion
			name="section-description"
```
Parce qu'en soit il y a aussi une description au niveau du champ